zerver/views/user_settings.py

```diff
@@ -260,6 +261,9 @@ def json_change_settings(
     if timezone is not None and user_profile.timezone != timezone:
         do_set_user_display_setting(user_profile, "timezone", timezone)
 
+    if enter_sends is not None and user_profile.enter_sends != enter_sends:
+        do_change_enter_sends(user_profile, enter_sends)
```

Comment:
We can likely do a follow-up PR to move this into the `property_types` framework.

Reply:
Will add this commit in the `RealmUserDefault` PR.

Reply:
@timabbott I had not done it before because `property_types` is used for display settings and we send event with type `update_display_setting` and I am not sure whether we can consider this setting as display setting.